src/shared/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,81 @@
+// this rule gives trouble on forward refed components
+/* eslint-disable react/display-name */
+import React from 'react'
```

Comment:
```suggestion
import React, { useState } from 'react'
```
again, just for consistency